src/vm/loaderallocator.cpp

```diff
@@ -1105,13 +1105,7 @@ void LoaderAllocator::Init(BaseDomain *pDomain, BYTE *pExecutableHeapMemory)
     // Take a page from the high-frequency heap for this.
     if (pExecutableHeapMemory != NULL)
     {
-#ifdef FEATURE_WINDOWSPHONE
-        // code:UMEntryThunk::CreateUMEntryThunk allocates memory on executable loader heap for phone.
-        // Reserve enough for a typical phone app to fit.
-        dwExecutableHeapReserveSize = 3 * GetOsPageSize();
```

Comment:
If we are changing the reserve size, does this result in a regression since we now must request more if it is greater than 3x?

Reply:
This change is an small improvement for typical .NET Core apps. Typical .NET Core apps use very little of ExecutableHeap and a lot more of other heaps.

The 3x multiplier here was a specific tweak for Silverlight on Windows Phone. It created several hundreds marshaled delegates during startup. The thunks for these delegates were allocated on ExecutableHeap ...